models/index.js

```diff
@@ -7,17 +7,60 @@ exports.fetchCategories = () => {
   });
 };
 
-exports.fetchReviews = () => {
-  const queryStr = `
-  SELECT reviews.review_id, reviews.title, reviews.designer, reviews.owner, reviews.review_img_url, reviews.category,
-  reviews.created_at, reviews.votes, COUNT(comment_id)::int AS comment_count FROM reviews
-  JOIN comments
-  ON reviews.review_id = comments.review_id
-  GROUP BY reviews.review_id
-  ORDER BY created_at desc;`;
-  return db.query(queryStr).then(({ rows: reviews }) => {
-    return reviews;
-  });
+exports.fetchReviews = (category, sort_by = "created_at", order = "desc") => {
+  const validSortByQueries = [
```

Comment:
you have added a lot of valid sort_by queries, but you've only tested that one of them works, there are ways to write tests to check they are all working, its not something that needs to be added but you certainly could.